systemd-swap

```diff
@@ -326,22 +332,22 @@ case "$1" in
       to_bytes(){ numfmt --to=none --from=iec "$1"; }
 
       systemd-notify "STATUS=Monitoring memory status..."
+
+      FSTYPE=$(get_fs_type "${swapfc_path}")
       # must exists before stat()
-      mkdir -p "${swapfc_path}"
+      mkdir -p $(dirname "$swapfc_path")
+      [ "$FSTYPE" == "btrfs" ] && btrfs subvolume create "${swapfs_path}" \
```

Comment:
This will try to create a btrfs subvolume even if there already exists a btrfs subvolume on `$swapfc_path`. Did you test if this creates an error?
Also typo here, it's `${swapfc_path}` not `${swapfs_path}`

Reply:
I've thought about that already and came to a conclusion that that's not a problem. Maybe I should suppress the error message? Or do you think adding an additional check here would be better?

Reply:
Just supress